tokio/tests/tcp_into_split.rs

```diff
@@ -82,6 +82,7 @@ async fn reunite() -> Result<()> {
 }
 
 /// Test that dropping the write half actually closes the stream.
+#[ignore]
```

Comment:
This is fixed in #2597.

Reply:
Nice! I've updated the PR description to add a reference to this issue. I kept the `#[ignore]` for the time being, otherwise the CI blocks on this test. Hopefully #2597 will be merged before this PR.